### ISSUE TITLE
Weeb Glove Broken

### ISSUE THREAD

**Issue by Kurfursten** [d3athrow/vgstation13#28054]
@SonixApache last year created ninja suits but forgot that weebs don't wear them, they have a black kimono, so they can't use their glove at all

this is why the power was in the glove!

**Comment by SonixApache** [comment]
I think fixing this would be as simple as adding a cell to the weeb glove. Will test some time soon.

**Comment by DeityLink** [comment]
Should be fixed now that weebs have been properly more or less replaced with actual space ninjas.

**Comment by Kurfursten** [comment]
No, this was after that. Weebs are still available as a bus button on the check antag panel (it will convert a ninja who has already spawned). Since this is still an option, it needs to be fixed.
